README.md

```diff
@@ -8,4 +8,28 @@
    1. Yelp is a system. As we've learned, systems are comprised of _entities_. List as many entities of Yelp as you can think of.
    2. Choose any **four** connected entities. First, in shorthand notes, describe the relationship between these interconnected entities.
    3. Use [draw.io](https://draw.io) to construct an ERD using Crow's Foot Notation for the four entities you selected.
-   4. In `erd.md`, describe your ERD in clear, concise sentences. Be sure to explain the reasoning behind each set of relationships in your diagram. Strong responses will use the phrases _has one_, _belongs to_, _has many_, and _belongs to many_. Be sure to export your diagram as a PNG file and include the photo in your markdown file.
\ No newline at end of file
+   4. In `erd.md`, describe your ERD in clear, concise sentences. Be sure to explain the reasoning behind each set of relationships in your diagram. Strong responses will use the phrases _has one_, _belongs to_, _has many_, and _belongs to many_. Be sure to export your diagram as a PNG file and include the photo in your markdown file.
+
+## Yelp Hypothetical Model
+
+**1. Possible Entities:**
+- Businesses
+- Users
+- Reviews
+- Photos
+
+**2. Quick Relationship Notes**
+- Users can own businesses (1 user, 1 business)
+- Users can post photos (1 user, many posts)
+- Businesses have many reviews (1 business, many reviews)
+- Users have one or no review for any number of businesses
+
+**3. EDR**
+
+![Image]('https://github.com/emtes/exit-ticket-7_6/blob/master/ticket-7-edr.png', 'Diagram')
+
+**4. EDR Description**
+- A user has a `1 to Many` relationship with a business where to own a business there must be at least 1 user, but a user can have no businesses.
```

Comment:
Correct. We call this a 1 to Optional Many. A user can have 0 or more businesses but a business must have exactly one user.